docs/docs/installation/alerts-reports.mdx

```diff
@@ -163,9 +163,9 @@ WEBDRIVER_OPTION_ARGS = [
 ]
 
 # This is for internal use, you can keep http
-WEBDRIVER_BASEURL = "http://superset:8088"
-# This is the link sent to the recipient. Change to your domain, e.g. https://superset.mydomain.com
-WEBDRIVER_BASEURL_USER_FRIENDLY = "http://localhost:8088"
+WEBDRIVER_BASEURL = "http://superset:8088/"
```

Comment:
Hey! Thanks for the fix! 🎉  just a comment, could we leave this as is and instead address the correct handling of both cases (with our without trailing /) in the get_url method?